llvm/lib/Transforms/IPO/LowerTypeTests.cpp

```diff
@@ -441,10 +441,6 @@ class LowerTypeTestsModule {
   // Cache variable used by hasBranchTargetEnforcement().
   int HasBranchTargetEnforcement = -1;
 
-  // The jump table type we ended up deciding on. (Usually the same as
```

Comment:
Confirming that passing this as arguments is just a drive-by cleanup?

Reply:
Correct